R/openapi-types.R

```diff
@@ -70,6 +70,24 @@ add_api_info_onLoad <- function() {
     as.character,
     location = c("query", "path")
   )
+  addApiInfo(
+    "date-time",
+    c("POSIXct", "POSIXt"),
+    "[^/]+",
```

Comment:
Could test for full format. From https://regex101.com/r/qH0sU7/1 :

(untested)
```suggestion
    # https://regex101.com/r/qH0sU7/1
    "^((?:(\\d{4}-\\d{2}-\\d{2})T(\\d{2}:\\d{2}:\\d{2}(?:\\.\\d+)?))(Z|[\\+-]\\d{2}:\\d{2})?)$",
```

Reply:
I've implemented a simpler regex and added some unit tests.

I tried your regex but got lost in trying to resolve all the optional components within a single group.

I felt sick when visiting https://stackoverflow.com/a/3143231/373321 and there were worse out there... I think what I have added is OK (you can review it...)

While here... I also wondered why `////d` is necessary here - not sure how we managed it, but we've made regex even more unreadable :)

Reply:
Am resolving this - but expecting it to be reviewed still